### ISSUE TITLE
KeyError getting table description

### ISSUE THREAD

**Issue by kinghuang** [awslabs/aws-data-wrangler#294]
**Describe the bug**

Calling [wr.catalog.get_table_description](https://aws-data-wrangler.readthedocs.io/en/latest/stubs/awswrangler.catalog.get_table_description.html#awswrangler.catalog.get_table_description) on a Glue table that doesn't have a description results in a KeyError exception being raised.

**To Reproduce**

```
awswrangler==1.5.0
```

1. Create a Glue table without a description.
2. Call `wr.catalog.get_table_description` on the table.

**Example**

The table `item` exists, but does not have a description.

![image](https://user-images.githubusercontent.com/27340/85229013-7e1c1700-b3a4-11ea-92f9-7f9378d40aba.png)

```python
>>> wr.catalog.get_table_parameters(database="sa-m2", table="item")
{'CrawlerSchemaDeserializerVersion': '1.0', 'CrawlerSchemaSerializerVersion': '1.0', 'UPDATED_BY_CRAWLER': 'sa-m2', 'averageRecordSize': '25', 'classification': 'parquet', 'compressionType': 'none', 'objectCount': '54', 'recordCount': '12349020', 'sizeKey': '174628122', 'typeOfData': 'file'}
>>> wr.catalog.get_table_description(database="sa-m2", table="item")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.8/site-packages/awswrangler/catalog.py", line 1670, in get_table_description
    desc: str = response["Table"]["Description"]
KeyError: 'Description'
```

**Comment by igorborgest** [comment]
@kinghuang Thank you for reporting it. We will fix and release a new version next week.

**Comment by igorborgest** [comment]
Hi @kinghuang  !

I've just fixed it in our development branch, and will be released soon on version `1.6.0`.

Could you test it, and give us some feedback?

To install the development branch:

```
pip install git+https://github.com/awslabs/aws-data-wrangler.git@dev
```

**Comment by kinghuang** [comment]
I just tried the development branch. It returns `None` now instead of raising an exception.

```python
>>> wr.catalog.get_table_description(database="sa-m2", table="item")
>>> print(wr.catalog.get_table_description(database="sa-m2", table="item"))
None
```

**Comment by igorborgest** [comment]
> I just tried the development branch. It returns `None` now instead of raising an exception.
> 
> ```python
> >>> wr.catalog.get_table_description(database="sa-m2", table="item")
> >>> print(wr.catalog.get_table_description(database="sa-m2", table="item"))
> None
> ```

Is this the desired behavior, right?

**Comment by kinghuang** [comment]
Yes, I believe `None` is an appropriate return value when there is no description.

**Comment by igorborgest** [comment]
Released on version [1.6.0](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.6.0)!

**Comment by igorborgest** [comment]
Hi @kinghuang  !

It is totally off-topic, but we are stating a [_"Who uses AWS Data Wrangler?"_](https://github.com/awslabs/aws-data-wrangler/tree/dev#who-uses-aws-data-wrangler) section. So feel free to add yourself if you want 😄 .
